venues/ICLR.cc/2019/Conference/python/groups.py

```diff
@@ -127,9 +125,15 @@
     'Paper/Reviewers/Unsubmitted': reviewers_unsubmitted_template,
 }
 
-def create_and_post(client, paper, template_key):
-    client.post_group(openreview.Group.from_json(
-        openreview.tools.fill_template(group_templates[template_key], paper)))
+def create_and_post(client, paper, template_key, members=[]):
+    group_to_post = openreview.Group.from_json(
+        openreview.tools.fill_template(
+            group_templates[template_key], paper))
+
+    if members:
```

Comment:
could you pass the members to the template so you need to add the if?

Reply:
just to clarify, are you just asking for us to remove the if statement, or are you asking us to add the members to the template object before turning it into a group?

Reply:
I thought that you could pass the members in the fill_template function but I see that is general function.